src/imitation/util/rollout.py

```diff
@@ -137,6 +137,7 @@ def min_episodes(n: int) -> GenTrajTerminationFn:
   """
   def f(trajectories: Sequence[Trajectory]):
     return len(trajectories) >= n
+  assert n >= 1
```

Comment:
weakly prefer this as first line in the method (validate input before you use it)